### ISSUE TITLE
Uncaught Error: Cannot find module '/lib-js/saml2'

### ISSUE THREAD

**Issue by khodayar** [Clever/saml2#150]
when I'm using 
`var saml2 = require('saml2-js');`
returns 
`Uncaught Error: Cannot find module '/lib-js/saml2'` in browser console

**Comment by DrogoNevets** [comment]
I am getting this error too

**Comment by AartiTanya** [comment]
@khodayar Hey how did you fix this? I have the same issue

**Comment by williamli** [comment]
I don't think this library works on the browser, but I ran into this problem on server side.

### TLDR;

I got around it by using `const saml2 = require('saml2-js/lib-js/saml2');` instead of `const saml2 = require('saml2-js');`

### Investigation

It seems the way index.js try to build a path to `/lib-js/saml2.js` is a little off.

**Comment by maazmk** [comment]
any news on the issue ?

**Comment by ruffrey** [comment]
For anyone struggling with this issue in typescript and webpack, the following seems to work. But it does require wrapping all of the saml2-js stuff you need.

```
import type saml2js from "saml2-js";
const samlLib = require("saml2-js/lib-js/saml2");

export type ServiceProvider = saml2js.ServiceProvider;
export type IdentityProvider = saml2js.IdentityProvider;
// export any other types needed
// do stuff with samlLib.*
```
